srcpkgs/gjs/patches/ignore-gtk3-subtest.patch

```diff
@@ -0,0 +1,26 @@
+this subtest produces FAIL.
```

Comment:
Drop this patch, it is a gtk+3 bug, not a gjs bug.
https://github.com/void-linux/void-packages/commit/fcc6d4c20a40ebbb5a5d245d3f06fbe49865816b